web-console/knikubevirt/link-between-PVC-VMdisk/link-between-PVC-VMdisk.md

```diff
@@ -0,0 +1,27 @@
+# Link between PVC and VM Disk
+
+In some cases, a disk is being backed by a Kubernetes Persistent Volume Claim (PVC).
+In the OpenShift Console UI, we list these PVCs out for the user to view and edit.
+In our Virtualization UI, we should consider how we might show that a Disk is backed by a PVC and then link over to that PVC.
+
+Also, we should consider if we want to link the other way: if the user happens to be looking at the PVCs, we would want to link to a VM making use of a PVC.
+
+## Linking from PVC to Disk
+
+From the PVC page, we add a link to the table only if CNV is installed (or maybe even only if there are VMs) that will link to the VM. 
+Under the "VM DISKS" column there will be a link to the Disk name.
+
+![linking pvc to disk](img/linking-pvc-to-disk.png)
+
+## The Selected VM Disk
+
+Since we can't link directly to a specific VM disk details and can only link to the disks list page, we can add the PVC details as another column to the table 
+We can also highlight (and later fade) the row of the selected VM disk.
+
+![pvc details](img/pvc-details.png)
```

Comment:
SOURCE VM looks like a resource badge.  is that what it is?

Reply:
@serenamarie125 I guess so, but I remember suggesting to add a label/icon that would indicate this is the source VM disk and then got a review It could be unclear and could mean that this is the bootable disk. That is why I took it off.

Reply:
Agreed this probably isn't a good use of the resource badge. Maybe we just stick to this row being highlight for a moment? It might be nice to see two mockups, one showing the highlight and one without to see if that would be more clear.